recipes/compile-multi-all-the-icons

```diff
@@ -1,3 +1,3 @@
 (compile-multi-all-the-icons
  :fetcher github :repo "mohkale/compile-multi"
- :files ("extensions/compile-multi-all-the-icons/*.el"))
+ :files ("extensions/compile-multi-all-the-icons/compile-multi-all-the-icons.el"))
```

Comment:
```suggestion
 :files ("extensions/compile-multi-all-the-icons/compile-multi-all-the-icons*.el"))
```

Reply:
@mohkale suggested that foo*.el be used, just like in your suggested change. I think you will have to revert the merge on the embark update to fall in line with that. In the meanwhile I'll update this one.